jeison.el

```diff
@@ -186,24 +186,24 @@ your needs, please consider declaring a separate class."
   (mapc (lambda (slot-descriptor) (jeison--set-path slot-descriptor nil))
         (jeison--class-slots class-name)))
 
-(defun jeison-read (type inp-thing &optional path)
-  "Read TYPE from ALIST-OR-JSON by the given PATH.
+(defun jeison-read (type inp-source &optional path)
```

Comment:
I feel bad for harassing you on that, but it is still not there. What is `inp`? Is it `input`? Then call it `input`, it's just two extra characters. But going further, what **is** `input-source`, it feels a bit tautological. Maybe we can call it `input-json` and then it kind of obvious.

Reply:
But if you call it `input-json`, you lose the alist part. :-)

Sure, if you want `input-source`, it can be `input-source`. `inp` has always been an abbreviation for input. And if you look around enough CL source, `thing` tends to be used when describing a polymorphic entity.

Reply:
`alist` is yet another representation of JSON, native Emacs Lisp JSON :-) So, I don't see this as a problem.